assets/component-cart.css

```diff
@@ -78,7 +74,7 @@ cart-items {
 }
 
 .cart__footer > * + * {
-  margin-top: 4rem;
+  margin-top: 6.5rem;
```

Comment:
This may be ok, but I'm not sure if apps can inject elements into the cart footer and will inherit this additional margin too. So it may be safer to only target elements that directly follow the cart note in the DOM:

``` css
@media screen and (max-width: 749px) {
  .cart__note + * {
    margin-top: 6.5rem;
  }
}
```